pkg/controller/render/render_controller.go

```diff
@@ -536,6 +538,11 @@ func getMachineConfigsForPool(pool *mcfgv1.MachineConfigPool, configs []*mcfgv1.
 	}
 	for idx, config := range configs {
 		if selector.Matches(labels.Set(config.Labels)) {
+			// Before adding a MC to the array of MCs for a pool, let's make sure it contains valid Ignition
```

Comment:
I only see `getMachineConfigsForPool()` as being called in the bootstrap path.

I think this may belong in `generateMachineConfig()`?

Reply:
i moved this a few times and you are absolutely correct.  will move again. ty!

Reply:
Done.